src/attack.c

```diff
@@ -1083,9 +1083,48 @@ apply_hosts_preferences_ordering (gvm_hosts_t *hosts)
     g_debug ("hosts_ordering: Sequential.");
 }
 
-static void
+static int
 apply_hosts_reverse_lookup_preferences (gvm_hosts_t *hosts)
 {
+  int hosts_excluded = 0;
+
+#ifdef FEATURE_REVERSE_LOOKUP_EXCLUDED
+
+  const char *exclude_hosts = prefs_get ("exclude_hosts");
```

Comment:
#ifdef FEATURE_REVERSE_LOOKUP_EXCLUDED
  ```suggestion
  int hosts_excluded = 0;
  const char *exclude_hosts = prefs_get ("exclude_hosts");
```
Without it, I get
```
attack.c:1089:7: error: unused variable ‘hosts_excluded’
```